integration-tests/api/__tests__/admin/price-list.js

```diff
@@ -43,6 +44,7 @@ describe("/admin/price-lists", () => {
       await adminSeeder(dbConnection)
       await customerSeeder(dbConnection)
       await productSeeder(dbConnection)
+      await priceListSeeder(dbConnection)
```

Comment:
nit(non-blocking): we should strive to avoid using "seeders" and instead use the factories, so we have isolated tests.

Reply:
Feel free to add `automerge` when you want this merged :